cloudbuild.yaml

```diff
@@ -0,0 +1,17 @@
+timeout: 3000s
+steps:
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211013-1be7868d8b
+    entrypoint: make
+    args:
+      - image-build
+      - image-push
+    env:
+    - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
```

Comment:
what does the project_id resolve to?

Reply:
to the k8s-infra-staging-kueue

Reply:
where is that specified?

Reply:
I suspect it's fed from https://github.com/kubernetes/test-infra/pull/25465/files via this image https://github.com/kubernetes/test-infra/tree/master/images/builder#usage